### PR TITLE
Boil lye into powder

### PR DESCRIPTION
**Pull request by ituluwituluwzev**
The recipe for making lye powder by boiling lye.

### CODE REVIEW

File: data/json/recipes/recipe_medsandchemicals.json [hunk 1]
```diff
@@ -1777,6 +1777,19 @@
...
+  "result": "lye_powder",
+  "category": "CC_CHEM",
+  "subcategory": "CSC_CHEM_CHEMICALS",
+  "skill_used": "cooking",
+  "difficulty": 3,
+  "time": 90000,
+  "autolearn": true,
+  "batch_time_factors": [ 80, 4 ],
+  "qualities": [ { "id": "BOIL", "level": 1 } ],
+  "tools": [ [ [ "water_boiling_heat", 3, "LIST" ] ] ],
```
Comment (Coolthulhu) [inline data/json/recipes/recipe_medsandchemicals.json]: 3 is just enough energy to bring a cup of water to boiling temperature. But the recipe involves boiling out 20 liters of liquid.

Comment (ituluwituluwzev) [reply data/json/recipes/recipe_medsandchemicals.json]: Thanks for pointing that out!

**Comment by angoddu** [comment]
What about using a dehydrator?

**Comment by ituluwituluwzev** [comment]
I don't think one can dehydrate solution. There is too much water.

**Comment by Amariithynar** [comment]
Milk is a solution of milk fats and water, and you can definitely dehydrate that; no reason you can't spread a liquid thin across the surface of a tray and dehydrate it.

**Comment by kevingranade** [comment]
Sources seem to disagree
[Don't do it](https://www.brainstuffshow.com/blogs/how-can-i-make-my-own-powdered-milk.htm)
[Only use pasteurized milk](https://www.survivalsullivan.com/make-powdered-milk-home/)

**Pull request merged**